platyPS.schema.md

```diff
@@ -23,7 +23,10 @@ It closely resembles output of `Get-Help`.
 
         ## SYNTAX
         // for each parameter set
+            ### Parameter Set Name
+            ```
             {{Output of Get-Command -Syntax}}
```

Comment:
I think it's unrelated to ParameterSets: in Add-Computer long syntax is not dues to the 2 parameter sets, but due to the number of parameters.
We should add support for line breaks inside syntax entry and can generate them, good point.